src/devices_models/devices/common/range_constraint.jl

```diff
@@ -22,28 +37,17 @@ where limits in constraint_infos.
 `` x = limits^{max}, \text{ for } |limits^{max} - limits^{min}| < \varepsilon ``
 
 `` limits^{min} \leq x \leq limits^{max}, \text{ otherwise } ``
-
-# Arguments
-* psi_container::PSIContainer : the psi_container model built in PowerSimulations
-* constraint_infos::Vector{DeviceRangeConstraintInfo} : contains names and vector of min/max
-* cons_name::Symbol : name of the constraint
-* var_name::Symbol : the name of the continuous variable
 """
-function device_range(
-    psi_container::PSIContainer,
-    constraint_infos::Vector{DeviceRangeConstraintInfo},
-    cons_name::Symbol,
-    var_name::Symbol,
-)
+function device_range(psi_container::PSIContainer, inputs::RangeConstraintInputsInternal)
     time_steps = model_time_steps(psi_container)
-    variable = get_variable(psi_container, var_name)
-    ub_name = middle_rename(cons_name, PSI_NAME_DELIMITER, "ub")
-    lb_name = middle_rename(cons_name, PSI_NAME_DELIMITER, "lb")
-    names = (get_name(x) for x in constraint_infos)
+    variable = get_variable(psi_container, inputs.variable_name)
+    ub_name = middle_rename(inputs.constraint_name, PSI_NAME_DELIMITER, "ub")
```

Comment:
should we make all of these use accessor functions?

Reply:
Accessor functions make lots of sense when you have (1) concrete structs that implement methods for abstract types and (2) a complex struct used across files where you may want to add locking or logging on setters and getters.  In this case the struct is defined in the same file and we probably won't ever have an abstract type.  On the other hand, accessors almost never causes future problems.  Just more effort.  If this doesn't persuade you, I'll add them.

Reply:
I will leave it up to you. I prefer the getters but if it's too much extra work it's not necessary.